hack/local-garden/cleanup

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+LABEL=${1:-local-garden}
+
+docker rm -f $(docker ps -f "label=$LABEL" -aq) 2&> /dev/null
```

Comment:
```suggestion
docker rm -f $(docker ps -f "label=$LABEL" -aq) 2&> /dev/null || true
```
see: https://unix.stackexchange.com/questions/212725/gnu-make-does-not-ignore-failed-command